src/octopeer-github/main/MainController.ts

```diff
@@ -13,10 +13,10 @@ class MainController implements OptionsObserver {
      */
     public start() {
         Logger.setDebug(); // TODO remove this on release
-        this.connectToContentScript();
-        Status.standby();
         Options.init();
         Options.addObserver(this);
+        Status.standby();
```

Comment:
Because I thought options should be initialized before connecting to the content script. You edited it in #117, I know, so it doesn't matter that much now :)